investd/config.py

```diff
@@ -30,10 +31,17 @@ def get_config_variables() -> dict[str, Any]:
     }
 
 
-log.info(
-    " - ".join([f"{name}: {value}" for name, value in get_config_variables().items()])
-)
-
-for conf_path in (SOURCE_BASE_PATH, PERSIST_PATH, REPORTS_PATH):
-    if not conf_path.exists():
-        os.makedirs(conf_path)
+def init_dirs() -> None:
+    """
+    Check if data dirs exist, create them otherwise.
+    """
+    config_vars = get_config_variables().items()
+    log.info(", ".join(f"{name}: {value}" for name, value in config_vars))
+    if not INVESTD_DATA.exists():
+        print(
+            f"INVESTD_DATA directory ({INVESTD_DATA}) doesn't exist. "
+            "Let me create it for you."
+        )
+    for conf_path in (INVESTD_SOURCES, INVESTD_PERSIST, INVESTD_REPORTS):
+        if not conf_path.exists():
+            os.makedirs(conf_path)
```

Comment:
how about exist_ok=True flag here instead of "if"?

Reply:
Indeed much better